core/javax.cache/src/main/java/org/wso2/carbon/caching/impl/clustering/ClusterCacheInvalidationRequestSender.java

```diff
@@ -56,6 +58,18 @@ public void entryUpdated(CacheEntryEvent event) throws CacheEntryListenerExcepti
      * there is an remove/update of the local cache in the current node.
      */
     public void send(CacheEntryEvent cacheEntryEvent) {
+        String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain(true);
+        int tenantId = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantId(true);
+
+        if (MultitenantConstants.INVALID_TENANT_ID == tenantId) {
```

Comment:
Not in this case, since this is a check of two integer number values. But if this is the convention, +1 for it.